.gitignore

```diff
@@ -31,5 +31,7 @@ pipeline.log
 .idea/
 
 # Tests
+coverage.out
 
-coverage.out
\ No newline at end of file
+# Binary
+# ./bin
```

Comment:
Is this intentionally un-ignored?

Reply:
Yes, I want to remove the binary from the tree.